### ISSUE TITLE
Export observed words

### ISSUE THREAD

**Issue by Hipapheralkus** [PortSwigger/param-miner#66]
It would be great if there was an option to see which observed words are there at the moment, and export them into a file (e.g. to be used in Intruder).

**Comment by albinowax** [comment]
Good idea! As a workaround you could achieve this with a Python script that writes the wordlist to a file
